modules/d/tools/dmd.lua

```diff
@@ -6,9 +6,10 @@
 
 	local tdmd = {}
 
-	local project = premake.project
-	local config = premake.config
-    local d = premake.modules.d
+	local p = premake
+	local project = p.project
+	local config = p.config
+    local d = p.modules.d
```

Comment:
The file was already using spaces, I feel like that change might be a bit unrelated to this PR. I'm happy to do up another PR if you'd like?